apps/web/pages/video/[uid].tsx

```diff
@@ -85,7 +160,24 @@ export default function JoinCall(props: JoinCallPageProps) {
     const response = recordingStartedEventResponse.parse(event);
     recordingId.current = response.recordingId;
   };
-
+  const handleTranscriptionStarted = () => {
+    //TODO - do we need anything here?
+  };
+  const handleTranscriptionStopped = async () => {
+    console.log("handleTranscriptionStopped is called");
+    if (!props.calAiCredential || !props.calAiCredential.key || !transcript.current) {
+      console.log("returning without calling the ai tool");
+      return;
+    }
+    //TODO
+    // send the transcript to the transcription meeting handler
```

Comment:
This relies on sendEmail; you'll need to do something similar to the above to get this to work through the backend, currently it's throwing an error it tries to send email from the browser.

```javascript
  fetch("/api/recorded-daily-video", {
      method: "POST",
      body: JSON.stringify(data),
    }).catch((err) => {
      console.log(err);
    });
    recordingId.current = null;
  };
```

Reply:
going to try this. thanks!